output/cloud/expv2/flush.go

```diff
@@ -59,6 +62,21 @@ func (f *metricsFlusher) flush(_ context.Context) error {
 	}
 
 	// send the last (or the unique) MetricSet chunk to the remote service
+	return f.push(msb)
+}
+
+// push sends the metric set to the remote service.
+// it also checks if the labels are discarded and logs a warning if so.
+func (f *metricsFlusher) push(msb metricSetBuilder) error {
+	for key := range msb.discardedLabels {
+		if _, ok := f.discardedLabels[key]; ok {
+			continue
+		}
+
+		f.discardedLabels[key] = struct{}{}
+		f.logger.Warnf("Label %s was discarded since it can't be used with the cloud output", key)
```

Comment:
```suggestion
		f.logger.Warnf("Tag %s was discarded since it is reserved for the Cloud operations.", key)
```
1. The user doesn't know the Label concept for k6. They set a Tag.
2. It isn't an issue of the Cloud output but more of the Cloud platform (I don't know how 
3. I think the _why_ isn't clear and the user could care otherwise it could be confused.